inc/extras.php

```diff
@@ -193,17 +195,18 @@ function eighties_excerpt_more( $more ) {
  *
  * @since  1.0.3
  * @access public
- * @param  string  $title
+ * @param  param $title For the function.
```

Comment:
@n8finch this should not be `param` but rather the type of param (e.g. string, array, object, mixed, et cetera).